.github/workflows/basics.yml

```diff
@@ -0,0 +1,62 @@
+name: BasicQA
+
+on:
+  # Run on all pushes and on all pull requests.
+  # Prevent the "push" build from running when there are only irrelevant changes.
+  push:
+    paths-ignore:
+      - '**.md'
+  pull_request:
+  # Allow manually triggering the workflow.
+  workflow_dispatch:
+
+jobs:
+  checkcs:
+    name: 'Basic CS and QA checks'
+    runs-on: ubuntu-latest
```

Comment:
The tests have got a warning:

> Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816

Presumably this won't matter as there is `shivammathur/setup-php@v2` being used for Basic and Test workflows already anyway?

Reply:
That warning is not directly related to the setup-php action. It is related to the jobs all using `runs-on: ubuntu-latest`.

For all intends and purposes, the ubuntu version shouldn't have any significant effect on the PHP environment (though, yes, that is providing the `setup-php` action will work on all), so I've been ignoring the warning.

Using `ubuntu-latest` is a shorthand way to always use the latest instead of a "fixed" version.
With Travis, there was a lot of differences between the images (`trusty`, `xenial`, `bionic`) and what they supported and it was more important to "fix" it to a specific version as it impacted which versions of PHP were available.

I don't have the impression the difference is as big on GH Actions, nor that it will impact the workflows we use.
Where it would, I'm expecting the `setup-php` action to fill in the difference.
Also see: https://github.com/shivammathur/setup-php/#cloud-osplatform-support

If and when GH actually switches over and the build would fail, we could always (temporarily) fix it to a specific `ubuntu` version until support for the next one has been repaired.

More than anything, it's about us not having to keep track of the specifics of the images and not having to update the name of the images used every so often and trusting that the actions we use will handle it.

Does that explain it well enough ?